pendulum/date.py

```diff
@@ -137,11 +137,11 @@ def day_of_year(self):
 
         :rtype: int
         """
-        k = 1 if self.is_leap_year else 2
+        k = 1 if self.is_leap_year() else 2
 
         return (
             (275 * self.month) // 9
-            - k * (self.month + 9) // 12
+            - k * ((self.month + 9) // 12)
```

Comment:
These parentheses are not needed since it's the same result either way. Unless I am missing something.